docs/millennium-os/manual/chapters/10_introduction.md

```diff
@@ -46,9 +51,9 @@ MillenniumOS uses the concept of features to enable and disable functionality. T
 * **Toolsetter** - disabled by default
 * **Touch Probe** - disabled by default
 * **Variable Spindle Speed Control** - enabled by default
-* **Spindle Feedback** - not implemented yet
+* **Spindle Feedback** - disabled by default
 
-Some of these features, like the Toolsetter and Touch Probe, require hardware support.
+Some of these features, like the Toolsetter, Touch Probe and Spindle Feedback, require hardware support.
```

Comment:
Oxford Comma? Might be a cleaner sentence to structure it as: "Some of these features require hardware support, such as: the ToolSetter, Touch Probe, and Spindle Feedback."

Reply:
@bdistin Rewrote that section slightly to take this into account in the feature list rather than calling them out individually. Does the new format make sense to you?